README.md

```diff
@@ -111,7 +111,7 @@ If you like this project, please don't forget to give it a ⭐️
 
 <!-- ACKNOWLEDGEMENTS -->
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
+## 🙏 Acknowledgments <a name="Nelson Sakwa"></a>
 
 > Give credit to everyone who inspired your codebase.
```

Comment:
- [ ] As suggested previously, Kindly make sure As in the project requirements, The README should contain in the "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).